pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/AuthenticationTest.java

```diff
@@ -1,44 +1,110 @@
 package org.broadinstitute.dsm;
 
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
+import static org.broadinstitute.dsm.TestHelper.cfg;
+import static org.broadinstitute.dsm.TestHelper.setupDB;
 
-import org.apache.http.HttpResponse;
-import org.broadinstitute.dsm.util.TestUtil;
+import org.broadinstitute.dsm.db.dao.user.UserDao;
+import org.broadinstitute.dsm.db.dto.user.UserDto;
+import org.broadinstitute.dsm.model.NameValue;
+import org.broadinstitute.dsm.model.patch.Patch;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.broadinstitute.dsm.util.UserUtil;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class AuthenticationTest {
+    public static final String UNIT_TESTER_EMAIL = "testUsersEmails.unitTesterEmail";
+    public static final String GP_UNIT_TESTER_EMAIL = "testUsersEmails.gpUnitTesterEmail";
+
+
+    @Before
+    public void first() {
+        setupDB();
+
+    }
+    @Test
+    @Ignore
```

Comment:
Can we get these classes and methods under test without turning off coverage?

Reply:
I am actually working on that too... changing configs to see if that fixes the errors

Reply:
@pegahtah - does this mean you're going to push changes in this PR that turn these tests back on or is that a future item?